pubspec.yaml

```diff
@@ -32,15 +32,15 @@ dependencies:
   flutter_secure_storage: ^9.0.0
   fluttertoast: ^8.0.8
   local_auth: ^2.1.7
-  flutter_map: ^5.0.0
-  geocoding: ^2.0.0
+  flutter_map: ^7.0.0
+  geocoding: ^3.0.0
   geolocator_android: ^4.1.9
   geolocator_apple: ^2.1.1+1
-  fl_chart: ^0.63.0
+  fl_chart: ^0.68.0
   latlong2: ^0.9.0
   intl: any
   flutter_form_builder: ^9.1.1
-  form_builder_validators: ^9.1.0
+  form_builder_validators: ^10.0.0
```

Comment:
The current Dart SDK version is 3.3.3.

Because nami depends on form_builder_validators >=10.0.0 which requires SDK version >=3.4.0 <4.0.0, version
  solving failed.

Reply:
Ich habe die packages alle upgraded, um mit Flutter 3.22 kompatibel zu sein. Das scheinst du noch nicht zu verwenden.